### ISSUE TITLE
Can't install Zen Mods

### ISSUE THREAD

**Issue by MarGustafsson** [zen-browser/desktop#2657]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

I'm trying to install some of the existing Zen mods, but every single one i look at only tells me:
`You need to have Zen Browser installed to install this theme`. 

I just installed Zen today, so not sure what could be causing this issue.

here is a screenshot of the problem:
![Screenshot 2024-11-05 at 09 50 41](https://github.com/user-attachments/assets/f309559b-8f4f-4602-a2aa-0e22f12ddbb4)

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

Version 1.0.1-a.17 (64-bit)

### What platform are you seeing the problem on?

macOS - aarch64

**Comment by MarGustafsson** [comment]
@dosu I must have missed that issue report i could not find it when i was searching.

however, the bug seems very much still active, since i installed Zen today and the bug was there, and the other issue was closed 3 weeks ago.

I am on the latest version, so it's not about that either.

TLDR: even with the closed issue for the same problem, the bug still occurs.

**Comment by MarGustafsson** [comment]
@dosu As this is happening on a fresh install, i dont think that anything should be setup incorrectly. i have done nothing else to the browser, but this is still happening.

I would also give a log output, but i have no idea how to get that, and i can't find any documentation on how to do so.

**Comment by MarGustafsson** [comment]
@dosu All messages in the console when looking at a mod:

`Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015. (Reason: CORS request did not succeed). Status code: (null).`

`None of the “sha512” hashes in the integrity attribute match the content of the subresource. The computed hash is “z4PhNX7vuL3xVChQ1m2AB9Yg5AULVxXcg/SpIdNs6c5H0NE8XYXysP+DGNKHfuwvY7kxvUdBeoGlODJ6+SfaPg==”. 64cdc40f-2366-4b5b-8bad-d0524682595e`

`Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015. (Reason: CORS request did not succeed). Status code: (null).`

`None of the “sha512” hashes in the integrity attribute match the content of the subresource. The computed hash is “z4PhNX7vuL3xVChQ1m2AB9Yg5AULVxXcg/SpIdNs6c5H0NE8XYXysP+DGNKHfuwvY7kxvUdBeoGlODJ6+SfaPg==”. 64cdc40f-2366-4b5b-8bad-d0524682595e`

`Uncaught Error: Minified React error #418; visit https://react.dev/errors/418 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    NextJS 29
2 fd9d1056-a2696c0ed8b60138.js:1:24670`

`Uncaught Error: Minified React error #418; visit https://react.dev/errors/418 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    NextJS 29
fd9d1056-a2696c0ed8b60138.js:1:24670`

`Uncaught Error: Minified React error #423; visit https://react.dev/errors/423 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    NextJS 29
fd9d1056-a2696c0ed8b60138.js:1:118349`

`The resource at “https://zen-browser.app/_next/static/media/a34f9d1faa5f3315-s.p.woff2” preloaded with link preload was not used within a few seconds. Make sure all attributes of the preload tag are set correctly.`

**Comment by MarGustafsson** [comment]
@dosu it's not a specific mod, it's all mods i've tried looking at.
I dont think i have any special firewall settings, I dont have a VPN on, so i don't think that's the issue either.

**Comment by MarGustafsson** [comment]
@dosu i disabled all the extensions i had gotten after importing, and then i could install the mods. 

i was using vue devtools, 1password, and ublock origin, if that helps.

**Comment by MarGustafsson** [comment]
just to add, i tried them individually, and specifically it seems that it's the 1Password extension that causes this to happen.
